docs/security/configuration/client-auth.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Client-based authentication
+parent: Configuration
+grand_parent: Security
+nav_order: 50
+---
+
+# Client-based authentication
+
+After obtaining your own certificates either from a certificate authority (CA) or by [generating your own certificates using OpenSSL](../generate-certificates), you can start configuring Elasticsearch to authenticate a user using a client certificate.
```

Comment:
Let's include information about why client certificate authentication is even an appealing option. Instead of a username and password, which can be the victim of some savvy guesswork and brute force, you've got the certificate and its key on a computer, and you've committed to keeping that computer safe. You might use this method with your Logstash machine or an ingest script. I don't know if "more" secure is the word, but I assume brute forcing a certificate is a few orders of magnitude more complicated than the typical password.